@djtimca Oh I assumed this was relevant since it has references to this PR: "Error: tests/components/aurora/test_config_flow.py:58:21: F821 undefined name 'ClientError'" from flake8.  Not sure why py38 was ok with that import missing, but it seemed related.

--- a/tests/components/aurora/test_config_flow.py
+++ b/tests/components/aurora/test_config_flow.py
@@ -2,6 +2,8 @@
 
 from unittest.mock import patch
 
+from aiohttp import ClientError
+
 from homeassistant import config_entries, data_entry_flow, setup
 from homeassistant.components.aurora.const import DOMAIN
 

